internal/repository/postgres/user/repo.go

```diff
@@ -114,6 +114,16 @@ func (r *Repository) GetByEmail(ctx context.Context, email string) (entity.User,
 	return u, nil
 }
 
+func (r *Repository) UserExists(ctx context.Context, email string) bool {
```

Comment:
you can take user by email, and check error

Reply:
@igkostyuk do we need to take a user from a db, or our user would be lightwight as it is now?
As this aproach uses the advantages of using SQL queries over ORMs

Reply:
lightweight user ? do you worry about performance issues? do we need user exist by id ?

Reply:
do we need userExist by id ? do we feel any performance issues ? lightwight user ?